charts/substra-backend/values.yaml

```diff
@@ -941,22 +941,23 @@ postgresql:
     database: *psql-database
   ## @skip postgresql.primary
   primary:
-    podSecurityContext:
-      runAsNonRoot: true
-      seccompProfile:
-        type: RuntimeDefault
-    containerSecurityContext:
-      allowPrivilegeEscalation: false
-      capabilities:
-        drop:
-        - ALL
```

Comment:
Was it not working there?

Reply:
Maybe related to the chart version update?

Reply:
Indeed, this has been lost in the various trials / rebase

The new `containerSecurityContext` is more restrictive, and since it's the only container in the pod, the `podSecurityContext` should not matter, but I'll add it again to be explicit.